duden/word.py

```diff
@@ -88,12 +96,19 @@ def article(self):
         """
         Word article
         """
+        # Find span with class "lemma__determiner"
+        article_element = self.soup.find('span', {"class": "lemma__determiner"})
+        if article_element is not None:
+            # remove soft hyphens "\xad" and return
+            return article_element.get_text().replace('\xad', '').strip() 
```

Comment:
Reusing the clear_text function from common.py would reduce code duplication.
```suggestion
            return clear_text(article_element.get_text())
```